tests/rbd/test_rbd_trash_operations.py

```diff
@@ -0,0 +1,276 @@
+"""RBD Image trash operations at scale
+Test Case Covered:
+CEPH-83582444 - [Baremetal] [ScaleTest] - Test all rbd trash operations at scale
+Steps:
+1) Create an RHCS cluster on baremetal nodes as specified in setup. Execute IOs to
+fill the cluster to a decent percentage
+2) Create 10 RBD pools and 100 images per pool parallely on this cluster
+3) Create snaps and clones for half of the images
+4) Run IOs on clones
+5) Move all images to trash parallely and verify
+6) Restore all images back from trash parallely and verify
+7) Trash purge all images parallely and verify
+8) Restore all images back and verify
+9) Enable rbd_move_to_trash_on_remove in client config and remove all images
+10) Restore all images back and verify
+11) Define a trash purge schedule and verify that all images adhere to this schedule
+12) Restore all images back and remove trash purge schedule and verify
+13) Disable rbd_move_to_trash_on_remove in client config and remove all images
+Pre-requisites :
+- need client node with ceph-common package, conf and keyring files
+- FIO should be installed on the client.
+Environment and limitations:
+- The cluster should have enough storage to create atleast 1000 images with 4G each
+- cluster/global-config-file: config/reef/baremetal/mero_conf.yaml
+- Should be Bare-metal.
+"""
+
+from ceph.parallel import parallel
+from ceph.rbd.initial_config import initial_rbd_config
+from ceph.rbd.utils import getdict
+from ceph.rbd.workflows.cleanup import cleanup
+from ceph.rbd.workflows.rbd import create_images, wrapper_for_image_ops
+from ceph.rbd.workflows.rbd_trash import purge_images_and_verify
+from ceph.utils import get_node_by_id
+from cli.rbd.rbd import Rbd
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def restore_and_create_new_image(
+    rbd,
+    pool,
+    pool_type,
+    pool_config,
+    image_config,
+    client,
+    multi_image_config,
+    images_without_snaps_and_clones,
+    test_ops_parallely,
+    **kw,
+):
+    """ """
```

Comment:
please add doc string.

Reply:
Will add as part of next PR since we need more info for all the steps implemented.